### PR TITLE
Add e-commerce screens.

### PR DESCRIPTION
**Pull request by AnkurChunekar**
No functionality has been added just static screens from e-com html css project.

Added:
- screens for login, signup, cart management, wish list, and homepage.
- Navbar, Footer and Card component.
- CSS files for respective pages and components.
- Images  for homepage screen and a product image.

CSB link: https://codesandbox.io/s/clever-worker-yr2530

### CODE REVIEW

File: src/pages/homepage/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+export default function Homepage() {
+  return (
+    <>
+       <Hero /> 
+      <main className="homepage-main-section">
+       <Categories /> 
+       <Products /> 
+      </main>
```
Comment (AtulPant2704) [inline src/pages/homepage/Homepage.jsx:16]: You should rename this as Main, all the components in react should start with a capital letter.

File: src/pages/productListing/ProductListing.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import "./ProductListing.css";
+import { dumbellPI } from "../../images/index";
+import { Card } from "../../components/index";
+
+export default function ProductListing() {
+  return (
+      <>
+    <main className="product-listing-page">
+      <aside className="filters-container p-xs">
```
Comment (AtulPant2704) [inline src/pages/productListing/ProductListing.jsx:9]: You should extract out this Filter container separately as a component, code would look much better.

**Review by AtulPant2704 — COMMENTED** [review]
Other then those two things rest all looks good, code is very clean and properly indented as well as commented.

File: src/pages/homepage/Homepage.jsx (hunk 1 above)
Comment (AnkurChunekar) [reply src/pages/homepage/Homepage.jsx:16]: Hey atul, thanks for your feedback but main here is a plain html element and not a React Component, am I missing something please revert back if it is so.

File: src/pages/productListing/ProductListing.jsx (hunk 2 above)
Comment (AnkurChunekar) [reply src/pages/productListing/ProductListing.jsx:9]: Yes, even i realised it, but this PR already has a lot of changes, hence thought of refactoring it in the next one.

**Review by RahullRawat — APPROVED** [review]

File: src/pages/homepage/Homepage.jsx (hunk 1 above)
Comment (AtulPant2704) [reply src/pages/homepage/Homepage.jsx:16]: Oh sorry! I missed that part.

File: src/pages/productListing/ProductListing.jsx (hunk 2 above)
Comment (AtulPant2704) [reply src/pages/productListing/ProductListing.jsx:9]: Okay no problem, I just wanted to inform you about the same.

**Review by AtulPant2704 — APPROVED** [review]

**Pull request merged**